### DISCUSSION TITLE
🛒 SFL Marketplace [PROPOSAL]

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4179] [Ideas]
This proposal explores the implementation of an in-game SFL marketplace to attain items to use in-game. Players will be able to use **SFL** to trade:

- Collectibles
- Wearables
- Bud NFTs
- Resources.

The marketplace will be accessible through our landing page and also in-game links (Trading Board). The public will be able to view it, however they must create an account to buy or sell.

<img width="777" alt="Screenshot 2024-07-25 at 6 04 50 PM" src="https://github.com/user-attachments/assets/cc5f6bac-2502-4ec6-8d47-85abebf69bc5">

The development of the marketplace will be split into V1, V2 & V3.

## Enter the Marketplace

The marketplace will be a one-stop shop for all things collectible in Sunflower Land. 

We will support **2 types of trades** in our marketplace:

1. Instant trade - players can execute these instantly as an off-chain trade (how our current P2P works). This will include any item's that were attained off chain (i.e. non supply Auction items). For instance, you can trade Mutant Chickens using the Instant Trade method.
2. On chain trade - this includes items which can only be transferred on chain (e.g. Kuebiko). The seller must sign a transaction to list the item and the buyer must submit a transaction to buy.

**Direct to farm 🧑‍🌾 **

You will be able to list items directly from your farm (no withdrawal needed). While listed, they will not be available for use.

Likewise, when you purchase an item it will be instantly available on your farm to use for your gameplay!

**Fees 🔥 **

All trades will require a 10% SFL fee. Instant trades will burn this fee (off chain). On chain trades, 5% will be transferred to Community Treasury & the other 5% to a revenue wallet to be exchanged for USDC at the team's discretion.

This will help generate funds for the project and the community treasury.

**Filters 📊 **

A player will be able to filter by the following:
- Decoration
- Utility (Crops, Gathering, Fishing, etc.)
- Search by name

We will sort from lowest price to highest. We will not show non-tradeable items or items that have not been listed yet. (see V2)

**Security 🔒 **

We want to avoid encouraging extractors and avenues where they can earn and withdraw SFL. Our P2P trading became a honeypot for malicious actors. Where possible we want to minimise this so it only appeals to real players.

The following mechanisms will be used to support such system:

1. Non VIP players get only 1 buy/trade per day
2. Items can only be traded once they are enabled for withdrawal
3. Resources continue to have min/max prices
4. Collectibles/Wearables can only be sold 1 at a time.
5. A player can only by the cheapest collectible/wearable listed

## Version 2 🚀 

Version 2 of the marketplace will be focussed around increasing engagement. Note that features in version 2 & 3 are not finalised and will depend on version 1 adoption.

Once we have the functional version live, we will focus on the following features:

**Trading Statistics 📈 **

We will surface a range of stats typically shown on a marketplace:
- Volume
- Owners

**Offers 🤝 **

Allow a buyer to make an offer on an SFT. A seller can then go to the marketplace and accept to fulfil the trade.

**Gamification & Points 🎮 **

Introduce a points system to reward players for trades. Points will be based on % of the SFL used. The **buyer** will receive these points for free.

You can exchange points for:
- Food
- Treasure Keys
- Raffles (Prize Ticket)

To encourage gathering points, we will incorporate a level system. When you reach X points, you will be upgraded to a new level that is visible in game.

We will introduce 15 levels, with a range of different prizes. For example:
- Level 1 - Basic Wearable + Coins
- Level 15 - Whale Onesie

## Version 3 🚀 

Version 3 focuses on a range of quality of life Improvements. This will factor in a lot of feedback from the previous versions. It also incorporates more collectible aspects of our gameplay.

**Upcoming Drops 📆 **

In the marketplace UI, we will showcase the upcoming drops as well at the Auction House.

This will be particularly useful to quickly see what is coming up & also entice the public to see the attainable items.

**Available Mints 🔨 **

Showcase which items are currently available and for how long. This will include seasonal items which expire soon.
- Megastore Items
- Desert Items

**History & Analytics 👴 **

A player will be able to deep dive into a single item on the marketplace:
- View pricing history
- Recent sales
- Advanced Marketplace Capabilities
- Sorting
- Advanced filters (e.g. filter by season launch)

# Additional Notes

**Hoarding Updates**

We will need to increase hoarding systems to ensure the system is flawless.

**Battle of the Bots** 

We will introduce some further detection on the trade limits - e.g identifying frequent buyers and limiting the amount of trades.

**Renting system** 

There have been recent issues with 3rd party rental systems. This could be a good opportunity to build a secure and reliable renting system in-game. For the scope of this proposal this will be discussed elsewhere.

**Legal Considerations**

As always, we will set the ethical benchmark for Web3 gaming. When building a marketplace we will ensure we don't promote speculation or shady tactics. With the recent DraftKings/Reignmakers debacle, it needs to be at the forefront of our decision making.

1. SFL is not a security/financial product. We have legal advice on this standing. The marketplace can use SFL since it is not considered a security.
2. We do not promote or sell items to have ROI
3. The marketplace does not encourage speculation - I.e. game assets are used for gameplay purposes primarily

When performing trades, we will need to provide clear modals & instructions around the above. All buyers/sellers should have a clear understanding of these. Our community is aligned on this, but we occasionally have new players with a different mindset.

## Feedback

As always, we appreciate all feedback from the players. These help shape these proposals and ensure we build something that the players want! 🤗

**Comment by brian-man** [comment]
> they must create an account

What does an account mean (a farm?)?

**Comment by brian-man** [comment]
> The marketplace does not encourage speculation

IMO, Bert's Obsession already does.

**Comment by snorri-snorri** [comment]
Why would I sell stuff for SFL when I get heavily taxed on withdrawing it? 
Is there some way around that, possibly introduce MATIC as a buy/sell option with no withdrawal tax? Unless there's already something I want in the game marketplace it's not really a good place to sell anything of value.

May I suggest a swap feature too, like we used on OS for buds

**Reply by adamhannigan** [reply]
That is a good point. I will share the ideas we have for this tomorrow (getting a bit late here).

We also need to consider someone who deposits SFL to attain an item. It would be a shame if they didn't get the item and then needed to pay withdraw tax. Ideally you shouldn't be taxed on anything that is deposited into the game.

**Reply by brian-man** [reply]
> We also need to consider someone who deposits SFL to attain an item. It would be a shame if they didn't get the item and then needed to pay withdraw tax. Ideally you shouldn't be taxed on anything that is deposited into the game.

A similar issue has applied to auctions for a long time now. (Depositing SFL or purchasing BB and then losing the auction being targeted.)

**Reply by adamhannigan** [reply]
Yes it isn't great.

I propose that SFL is **conditionally taxed** on withdraw. Only SFL that was earned in game should have the tax applied.

SFL that will be taxed on withdraw:
- Delivery SFL
- Goblin Exchange SFL
- In-game rewards (faction leaderboard, special events etc.)
- P2P resource trading

SFL that will not be taxed on withdraw:
- Deposited SFL
- On chain airdrops (e.g. Fan Art + Raffle prizes)
- SFL from sold collectibles
- SFL from sold wearables
- SFL from bud NFTs

Challenge: Will players be able to bypass SFL withdraw fees by buying an item and selling it instantly for SFL? Instead of paying a withdraw tax of 20%, they would just take the 10% marketplace tax.

It seems likely - we can figure out some interesting solutions to this....or just let it be. It would definitely help with engagement + launching the feature.

**Reply by NotKeen** [reply]
@adamhannigan Maybe get rid of the withdraw tax and tax the in-game SFL sources instead (like an income tax / "goblin tax"):

- Delivery SFL: might be okay to not be taxed, since it is auto-limited already (due to number of weekly deliveries)
- Goblin Exchange: You can put a tax there
- In-game rewards: Tax if lots of SFL is created there, might be okay to leave untaxed if the minted SFL is small there
- P2P trading: You could argue it is already taxed by the trading fee, maybe increase it.
- Direct selling of crops: Tax it

There could be different levels of taxation, maybe based on Bumpkin level or other achievements / signs of commitment which extractors might be less likely to achieve.

**Comment by elvennoia** [comment]
on one side, this will remove arbitrage monopoly of few people that prey on the offer and floor selling in opensea. they've been ruining the floor of all sft in marketplaces.

on other side, it does create added tax for sfl. sft has been almost independent from sfl fluctuation in some sense, but upon implementation it will fluctuate more.

so this marketplace will grately benefit buyer, but inexpense of the added burden to seller. will it have enough usage tho...

**Comment by ShinKan42** [comment]
The current market involves Emblems in play, please keep in mind how to deal with the Emblem market:
- either integrating it into the marketplace altogether
- or have it separated with the old UI & system

**Reply by adamhannigan** [reply]
Definitely - we can fix this separate to this proposal 👍 

https://github.com/orgs/sunflower-land/projects/45/views/1?pane=issue&itemId=73625042

**Comment by LucasAG95** [comment]
I would really like to have a marketplace in SFL, but honestly, from the proposal, it doesn’t seem very interesting to sell an SFT there because it's fees on top of fees. The person selling will have to pay the sale fee and then the fee to withdraw the SFL.

The idea of an SFL marketplace is good as an additional option. I just really hope you don’t make SFTs impossible to withdraw, because that defeats the purpose of web3, and I don’t want to see it turn into something like Bombcrypto.

I understand that as a company you need to profit, but the idea of you receiving SFL as payment makes me wary, just like many other games where devs received payments that way.

But if you want to charge fees on sales, think of something that truly makes players want to sell within the game, something that makes it worthwhile, and that they are not obligated to sell only within the game. For instance, every time you sell an SFT within the game, the 10% sale fee could be converted into “Sale Emblems.”

With these emblems, you could create a new auction category for the season or an exclusive SFT auction each season. But include SFTs that are worth it, that players would want to have.

For example, take the Lemon Shield with a supply of 250.

- 100 for Season Emblems
- 50 for SFL
- 50 for BB
- 50 for Sale Emblems

Or an exclusive item, such as:
SFT = Coin Frame - Supply 50 - Buff: 25% more coins in Betty

- 50 for Sale Emblems

**Reply by adamhannigan** [reply]
Great feedback 👌 

I like the idea of framing the fee with some reward - it definitely lessens the blow.

On the SFL side we would not hold any SFL token. The fees would likely be moved into a single asset vault/liquidated. I don't think the team should hold a reserve of SFL that we profit off speculation - this wouldn't encourage long term thinking.

I agree on your "fees on fees" comment. I've added some comments above on this 😄 

https://github.com/sunflower-land/sunflower-land/discussions/4179#discussioncomment-10250064

**Reply by LucasAG95** [reply]
"One thing you could also do is convert the fee into ‘BBs,’ albeit at a higher value. Here’s an example:

Item: Turbo Sprout - $300
The 10% fee would amount to $30.
Take that $30 and divide it by the value assigned to each ‘BB.’ Let’s assume each ‘BB’ is worth $0.5.
So, $30 divided by $0.5 equals 60 ‘BBs.’
In the end, the person who sold the Turbo Sprout would receive $270 in cash (after deducting the fee) and an additional 60 ‘BBs.’"

**Comment by vitt0** [comment]
I think that % that is taxed would be good if scales according with the price of the item that is being listed, mainly for the on-chain trades that I Imagine are going to be the most valuable in terms of FIAT, otherwise I think it would be kinda hard to see someone selling a exclusive item like could be a Kuebiko that is around 800$, paying a flat 10% over that is a decent amount and most of the players would prefer to take it to Opensea were they only have to pay 2,5% tax from OS and if they want they can set the royalties tax to 0%.

So the idea would be to have a similar model to the withdraw tax that $SFL has (that goes from 30% to 10% according to the quantity that they are withdrawing), so it could go from 10% as base to maybe 3 or 5% for listings that have a higher price.

**Comment by LuksmSFL** [comment]
Hello

Through not directly linked to this proposal, I would like to make a remark about resources/crops marketplace ingame : 

I work as a trader in my daily life, so my focus here will be about the economical loops built this year ingame :

It seems to me, imo, that the dual loop coin/SFL, which is impressively well thought, is made for a purpose : accelerating the circulation of money, and so multiply fees, which is, again, and still imo, very well thought. Congrats for the past hard work. It's a very pleasant economical loop to analyse. Very inspiring.

Sadly such a beautiful system appears to me as being forced to "walk on one leg only".

I explain myself : 

At the moment, the P2P market only permit two types of orders : 

1. Seller limit order
2. Buyer market order

It would be a great improvement (again imo), and if the will of the dev team is really to accelerate the circulation of money inside the economical loop, to "mirror" the actual market, with the two other possibles orders : 

3. Buyer limit order
4. Seller market order

With the same features (3 max, daily limit, ...)

Please think about it : At the moment, the game is giving "confort" to seller (possibility to choose selling price to limit order) while it force buyers with strategy to "waste time" (imagine a player ready to buy ANY gold under 2 SFL, he has to check the market all day around in order to wait such an order to appear and be the first one on it). Said differently : Seller can sell during "passive time" while buyers need to buy spending "active time" in addition to their buying price. Since any cryptogame is a trading pair "active time spent ingame" against crypto, such an asynchronous market is currently helping prices to stay "lower" than they should -imo again).

I know you are currently planning mass adoption via Apple, and I think that before that it would be great to change the "strong hand" and give it to buyers.

What you really want here (always imo), is new capital entering the game, making the deflationist SFL raise in value. 

In order to do so, you want people having AT LEAST as much ease to buy things that you wanted, in the early days, sellers to be able to sell easily in order to promote the play to earn dimension.

- New whales being able to place limit buying orders in order to get filled first and rush
- New beginners being able to instant sell by filling such orders in order to build their path faster

That's the two main components of a symetric marketplace that are not currently possible in the current asynchronous P2P marketplace.

That's what I think and wanted to share with you.

Have a nice day :)

**Comment by SFLAustinHarris** [comment]
Great proposal team! 

My only critique is to be careful of devaluing cosmetics by supplying the market with similar products. I don't think there should be a whale onesie if there is a whale hat, the same way I don't think there should be a shark hat if there is already a shark onesie.

By adding in a "whale onesie", it feels 
1. like you're taking away from the exclusivity of a previously unique item and 
2. somewhat lazy considering how talented and creative the art team is. 

I know that the whale onesie is just a placeholder for this proposal, but I just wanted to bring awareness so that we do not start a trend of pushing out non-unique cosmetics. The team removed the halvening mechanic so as not to devalue utility SFTs, and I think the same thoughtfulness should be used here as well.

*Potential solution*
Currently, the only way to get access to Hourglass utility items (50% cooking speed for 4 hours, -25% flower growth, etc) and high XP food are by paying marks in the faction shop. This means that you must join a faction in order to earn those marks. But what if a player does not want to join a faction and just wants the utility items and high XP food? There could be an opportunity to reward leveling up in the marketplace with items similar to the faction shop. 

Thanks for listening! :)

**Reply by adamhannigan** [reply]
Thanks for your feedback @SFLAustinHarris 

I agree we don't want similar cosmetics - I was mainly using it for an example. It brings up a good point that we will want custom and desirable items to attain.

Do you have any suggestions for cosmetics that would fit the theme of trading?

Using the temporary hourglasses also feels like a great reward 👏

**Reply by SFLAustinHarris** [reply]
Thanks for the question! After some pondering, I think the system could be set up how the faction wearables are. So once you reach a certain level, you unlock a new piece of the total set. The theme would be "broker" and designated wearables could be either +X trades per day,  -X% trading fee or +X% points per trade. So it could look something like

Level 1: Coins, Food, Treasure Key

Level 2: **Background**
Trading Floor (+1 trade per day): A bustling stock exchange backdrop with digital tickers and trading screens.

Level 5: **Hats/Headware**
Broker’s Top Hat : A classic top hat with a dollar sign

Level 7: **Mask**
Broker's Monocle: A classic pair of glasses

Level 10: **Secondary Tool**
Broker's Briefcase (+1 points per trade): A sleek briefcase with money bursting out of the sides.

Level 12: **Suit**
Broker's Suit: A high-end suit with a sleek design.

Level 14: **Shoes**
Broker's Loafers: Polished leather loafers 

Level 15 **Primary Tool**
Golden Calculator (-1% trading fee): A shiny calculator with elaborate designs.

And then of course people can spend those points on high XP food that fit into the broker theme like:
-Revenue Ramen: A hearty bowl of ramen with ingredients that resemble trading symbols and currency.
-Market Mixer Salad: A fresh salad with green and red tickers as lettuce
-Gold Bar Granola: A crunchy granola bar that looks like gold bars.
-Capital Cake: An elaborate cake decorated with edible gold leaf and currency notes.
-Portfolio Pizza: A pizza sliced into a pie chart
-Trader’s Tacos: Flavorful tacos with ingredients

**Reply by adamhannigan** [reply]
All this talk of Trader's Tacos is making me hungry! 😅

**Comment by Ark-SFL** [comment]
Olá, Adam! Espero que esteja bem.

Looking forward to the arrival of this market! A complete game changer, just like the revamp of skills.

But I had a doubt: couldn't the "Instant trade" method, which is based on a type of off-chain trading, be made from Item vs Item?

`Example: WTS > Green Amulet = Kuebiko
`
This means: I sell GA or trade for Kuebiko.
In MMO games this type of negotiation between items vs. items is very common, wouldn't it be possible to have it too? In this sense, players negotiate values ​​among themselves.

Like that:
![image](https://github.com/user-attachments/assets/9312c5fe-8a91-4389-bed0-a37ce58a8456)

**Reply by adamhannigan** [reply]
@Ark-SFL Nice suggestion 👌 Apologies for the delay, I missed the notification.

I agree that trade negotiation is a fun feature to include. For our initial version though, we are going with a simpler approach to the trading. 

This could be a nice addition for swapping items amongst players 👍

**Reply by Ark-SFL** [reply]
I totally understand, Adam! I hope we see this feature soon, I can't wait.

And of course, I hope this suggestion is also considered in the future. Thanks!

**Comment by heatedlime** [comment]
Can you please add another decimal place in the marketplace for item price? For example, soybeans can listed as .00719, but show up as .0071

**Comment by heatedlime** [comment]
Can you PLEASE add the ability to list more than 3 listings at a time? I love trying to arbitrage resource sales by buying low, and then selling higher weeks later, but it’s tough to do so with only 3 listings allowed at a time

**Comment by heatedlime** [comment]
> This proposal explores the implementation of an in-game SFL marketplace to attain items to use in-game. Players will be able to use **SFL** to trade:
> 
> * Collectibles
> * Wearables
> * Bud NFTs
> * Resources.
> 
> The marketplace will be accessible through our landing page and also in-game links (Trading Board). The public will be able to view it, however they must create an account to buy or sell.
> 
> <img alt="Screenshot 2024-07-25 at 6 04 50 PM" width="777" src="https://private-user-images.githubusercontent.com/11745561/355013979-cc5f6bac-2502-4ec6-8d47-85abebf69bc5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyNjgxNTQsIm5iZiI6MTcyNzI2Nzg1NCwicGF0aCI6Ii8xMTc0NTU2MS8zNTUwMTM5NzktY2M1ZjZiYWMtMjUwMi00ZWM2LThkNDctODVhYmViZjY5YmM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDEyMzczNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2YTA0YWQyZGUxODExNWViNGE0MjFkZDg1ZmEzMDM1MDliYjUyMzZiYjgxNTdkMmZkYmExOTlmZDYwNTBhMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.yXr9wz235OlY-aNIh8eY62WaVQ5a_TeOO4MIQODpnNc">
> The development of the marketplace will be split into V1, V2 & V3.
> 
> ## Enter the Marketplace
> The marketplace will be a one-stop shop for all things collectible in Sunflower Land.
> 
> We will support **2 types of trades** in our marketplace:
> 
> 1. Instant trade - players can execute these instantly as an off-chain trade (how our current P2P works). This will include any item's that were attained off chain (i.e. non supply Auction items). For instance, you can trade Mutant Chickens using the Instant Trade method.
> 2. On chain trade - this includes items which can only be transferred on chain (e.g. Kuebiko). The seller must sign a transaction to list the item and the buyer must submit a transaction to buy.
> 
> **Direct to farm 🧑‍🌾 **
> 
> You will be able to list items directly from your farm (no withdrawal needed). While listed, they will not be available for use.
> 
> Likewise, when you purchase an item it will be instantly available on your farm to use for your gameplay!
> 
> **Fees 🔥 **
> 
> All trades will require a 10% SFL fee. Instant trades will burn this fee (off chain). On chain trades, 5% will be transferred to Community Treasury & the other 5% to a revenue wallet to be exchanged for USDC at the team's discretion.
> 
> This will help generate funds for the project and the community treasury.
> 
> **Filters 📊 **
> 
> A player will be able to filter by the following:
> 
> * Decoration
> * Utility (Crops, Gathering, Fishing, etc.)
> * Search by name
> 
> We will sort from lowest price to highest. We will not show non-tradeable items or items that have not been listed yet. (see V2)
> 
> **Security 🔒 **
> 
> We want to avoid encouraging extractors and avenues where they can earn and withdraw SFL. Our P2P trading became a honeypot for malicious actors. Where possible we want to minimise this so it only appeals to real players.
> 
> The following mechanisms will be used to support such system:
> 
> 1. Non VIP players get only 1 buy/trade per day
> 2. Items can only be traded once they are enabled for withdrawal
> 3. Resources continue to have min/max prices
> 4. Collectibles/Wearables can only be sold 1 at a time.
> 5. A player can only by the cheapest collectible/wearable listed
> 
> ## Version 2 🚀
> Version 2 of the marketplace will be focussed around increasing engagement. Note that features in version 2 & 3 are not finalised and will depend on version 1 adoption.
> 
> Once we have the functional version live, we will focus on the following features:
> 
> **Trading Statistics 📈 **
> 
> We will surface a range of stats typically shown on a marketplace:
> 
> * Volume
> * Owners
> 
> **Offers 🤝 **
> 
> Allow a buyer to make an offer on an SFT. A seller can then go to the marketplace and accept to fulfil the trade.
> 
> **Gamification & Points 🎮 **
> 
> Introduce a points system to reward players for trades. Points will be based on % of the SFL used. The **buyer** will receive these points for free.
> 
> You can exchange points for:
> 
> * Food
> * Treasure Keys
> * Raffles (Prize Ticket)
> 
> To encourage gathering points, we will incorporate a level system. When you reach X points, you will be upgraded to a new level that is visible in game.
> 
> We will introduce 15 levels, with a range of different prizes. For example:
> 
> * Level 1 - Basic Wearable + Coins
> * Level 15 - Whale Onesie
> 
> ## Version 3 🚀
> Version 3 focuses on a range of quality of life Improvements. This will factor in a lot of feedback from the previous versions. It also incorporates more collectible aspects of our gameplay.
> 
> **Upcoming Drops 📆 **
> 
> In the marketplace UI, we will showcase the upcoming drops as well at the Auction House.
> 
> This will be particularly useful to quickly see what is coming up & also entice the public to see the attainable items.
> 
> **Available Mints 🔨 **
> 
> Showcase which items are currently available and for how long. This will include seasonal items which expire soon.
> 
> * Megastore Items
> * Desert Items
> 
> **History & Analytics 👴 **
> 
> A player will be able to deep dive into a single item on the marketplace:
> 
> * View pricing history
> * Recent sales
> * Advanced Marketplace Capabilities
> * Sorting
> * Advanced filters (e.g. filter by season launch)
> 
> # Additional Notes
> **Hoarding Updates**
> 
> We will need to increase hoarding systems to ensure the system is flawless.
> 
> **Battle of the Bots**
> 
> We will introduce some further detection on the trade limits - e.g identifying frequent buyers and limiting the amount of trades.
> 
> **Renting system**
> 
> There have been recent issues with 3rd party rental systems. This could be a good opportunity to build a secure and reliable renting system in-game. For the scope of this proposal this will be discussed elsewhere.
> 
> **Legal Considerations**
> 
> As always, we will set the ethical benchmark for Web3 gaming. When building a marketplace we will ensure we don't promote speculation or shady tactics. With the recent DraftKings/Reignmakers debacle, it needs to be at the forefront of our decision making.
> 
> 1. SFL is not a security/financial product. We have legal advice on this standing. The marketplace can use SFL since it is not considered a security.
> 2. We do not promote or sell items to have ROI
> 3. The marketplace does not encourage speculation - I.e. game assets are used for gameplay purposes primarily
> 
> When performing trades, we will need to provide clear modals & instructions around the above. All buyers/sellers should have a clear understanding of these. Our community is aligned on this, but we occasionally have new players with a different mindset.
> 
> ## Feedback
> As always, we appreciate all feedback from the players. These help shape these proposals and ensure we build something that the players want! 🤗

I frequently buy resources and flip them when the prices are higher.  The current way things are really works against me. Specifically hoarder limits and sale listing limits. 

I typically have to sync to the chain AT LEAST twice a day. I would suggest you at least double all hoarder limits for resources. 

Also, only having the ability to have 3 active listings sucks. The minimum I’d like is 10.

**Comment by NotKeen** [comment]
I really like the renting idea. I've wished for a long time the two functions of NFTs would be seperated:

1. The functional use of them, be it a boost or decorating your farm vs.
2. The speculation with or collection of NFTs while being able to rent them out and gain some "yield" if you just want to own them and not feel "forced" to actually use them

Classical split into workers and owners :)

**Reply by cryswerton** [reply]
Yeah, it would be great to rent NFTs.

**Comment by cryswerton** [comment]
Do non-VIP players still have only one buy/sell per day on the marketplace?

**Comment by eliasSFL** [comment]
Closing this discussion since marketplace has already been live for a year
